hist/hist/src/TFormula_v5.cxx

```diff
@@ -2685,10 +2686,10 @@ Double_t TFormula::EvalParOld(const Double_t *x, const Double_t *uparams)
 {
    Int_t i,j;
    // coverity[uninit] the tab value of tab is guaranteed to be set properly by the control flow.
-   Double_t tab[kMAXFOUND];
-   const char *stringStack[gMAXSTRINGFOUND];
+   Double_t tab[kMAXFOUND] = {0};
```

Comment:
The last time I checked, the control flow insure it was initialized before use.  Given that this a deprecated class, I would not change.

Albeit, it looks like the comment needs to be updated for the current version of coverity and/or the report be flagged as false positive.